code/modules/projectiles/guns/projectile/javelin.dm

```diff
@@ -107,7 +107,11 @@
 /obj/effect/overload/Process()
 	for(var/turf/T in trange(2, get_turf(src)))
 		for(var/mob/living/L in T)
-			L.electrocute_act(shock_damage * (FAST_PROCESS_INTERVAL * 0.1), src, 1, def_zone = BP_OVERALL)
+			if(iscarbon(L))
+				var/mob/living/carbon/C = L
+				C.electrocute_act(shock_damage * (FAST_PROCESS_INTERVAL * 0.1), src, 1, BP_OVERALL)
+			else
+				L.electrocute_act(shock_damage * (FAST_PROCESS_INTERVAL * 0.1), src, 1)
```

Comment:
this is a messy solution
if the living version of the proc doesn't have that extra parameter, just add it

Reply:
I have other way for fixing. Now do it.

Reply:
Ready